test/stdlib/CreateObject.test.ts

```diff
@@ -0,0 +1,16 @@
+const { AssociativeArray, BrsObjects, CreateObject, BrsInvalid } = require("../../lib/brsTypes");
+const BrsError = require("../../lib/Error");
+
+describe("CreateObject", () => {
+    beforeEach(() => BrsError.reset());
+
+    it("creates a new instance of associative array", () => {
+        let obj = CreateObject("roAssociativeArray");
+        expect(obj).toEqual(new AssociativeArray([]));
```

Comment:
How does equalTo work here?  Where is the comparison implemented?  Curious for future objects we add.

Reply:
That comes [from jest](https://jestjs.io/docs/en/expect#toequalvalue), and performs a deep equality check between two JavaScript objects.